### ISSUE TITLE
Exported images contain alpha channel and cannot be uploaded to App Store

### ISSUE THREAD

**Issue by PrestonL** [onmyway133/IconGenerator#16]
We need a new release:

ERROR ITMS-90717: "Invalid App Store Icon. The App Store Icon in the asset catalog in 'BoogerBoomCafe.app' can't be transparent nor contain an alpha channel."

**Comment by onmyway133** [comment]
@PrestonL ja, you need to recheck your icon files before using Icon Generator
